src/Sylius/Bundle/InstallerBundle/Command/InstallSampleDataCommand.php

```diff
@@ -56,8 +56,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $doctrineConfiguration->setSQLLogger(null);
 
         $commands = array(
-            'doctrine:fixtures:load'       => array('--no-interaction' => true),
             'doctrine:phpcr:fixtures:load' => array('--no-interaction' => true),
+            'doctrine:fixtures:load'       => array('--no-interaction' => true),
```

Comment:
I'm referencing image documents from Phpcr fixtures (in `ORM\LoadImagesData`) so I need to load them first.